pages/swagger.tsx

```
@@ -1,5 +1,28 @@
+            transform: 'translate(-50%, -50%)',
+          }}
+        />
+      </Container>
+    ),
+  }
+);
 
 export default function Swagger(): React.ReactElement {
-  return <ReactSwagger />;
-}
+  return <SwaggerComponent />;
```

ESLint

[error] 'SwaggerComponent' is not defined.

(react/jsx-no-undef)